tests/phpunit/api/v4/Api4TestBase.php

```diff
@@ -46,7 +46,8 @@ public function __construct($name = NULL, array $data = [], $dataName = '') {
   }
 
   public function setUpHeadless(): CiviEnvBuilder {
-    return Test::headless()->apply();
+    // TODO: search_kit should probably be part of the 'headless()' baseline.
+    return Test::headless()->install(['org.civicrm.search_kit'])->apply();
```

Comment:
Yeah, this move seems like a good balance.

Coleman linked this in another discussion, where it had been left as an open-question about _how far_ up to move the `install(...search_kit...)` statement. The tension in that question: Search Kit is a "required extension", but that concept is contradictory:

* As a "required" thing, it should be enabled by default in all integration tests (to make them more representative of the real runtime - and to avoid redundant boilerplate).
* As an "extension", it should be a discrete thing with clear boundaries (and you can't enforce/test those boundaries if it's always-on).

That's a tricky line. The current patch is a decent balance (*enabling it for a large block of tests where they're likely to be needed -- without making it totally required or affecting 3rd party suites*).